tools/dependency_summary.py

```diff
@@ -572,6 +572,51 @@
             "fixup": "https://raw.githubusercontent.com/myrrlyn/wyz/main/LICENSE.txt",
         },
     },
+    "proc-macro-error": {
+        "license": {
+            "check": "MIT OR Apache-2.0"
+        },
+        "repository": {
+            "check": "https://gitlab.com/CreepySkeleton/proc-macro-error"
+        },
+        "license_file": {
+            "check": None,
+            "fixup": "https://gitlab.com/CreepySkeleton/proc-macro-error/-/raw/master/LICENSE-APACHE"
+        }
+    },
+    "proc-macro-error-attr": {
+        # This is path dependency in `proc-macro-error` and uses
+        # it's same license
+        "license": {
+            "check": "MIT OR Apache-2.0"
+        },
+        "repository": {
+            "check": "https://gitlab.com/CreepySkeleton/proc-macro-error"
+        },
+        "license_file": {
+            "check": None,
+            "fixup": "https://gitlab.com/CreepySkeleton/proc-macro-error/-/raw/master/LICENSE-APACHE"
+        }
+    },
+    # Based on https://github.com/Alexhuszagh/minimal-lexical/blob/main/LICENSE.md
+    # the libraries is licensed as dual licensed, however a portion of the
+    # source code is licensed under BSD-3-Clause. The portion is only
+    # used if the code uses the `compact` feature. However, tracking down
+    # where it's used in the tree, it's used by
+    # https://github.com/Geal/nom/blob/294ffb3d9e0ade2c3b7ddfff52484b6d643dcce1/Cargo.toml#L38
+    # which does **not** use the `compact` feature so we use to the Apatche license.
```

Comment:
I'm gonna need some more license experienced eyes on this 😅

Reply:
typo: s/to the Apatche/the Apache/

I can't claim to have license experienced eyes, but I think your conclusion looks perfectly reasonable. It does sound like you are saying that license file isn't quite correct, so maybe opening an issue on their repo would be nice, but I don't think it's required!